### ISSUE TITLE
NativeScript Angular 2

### ISSUE THREAD

**Issue by HoangJK** [triniwiz/nativescript-image-cache-it#9]
Hello,

I have a question. How to use your plugin with NativeScript Angular 2?

Thanks.

**Comment by triniwiz** [comment]
Add the following to your `main.ts`

```
import { registerElement } from "nativescript-angular/element-registry";
registerElement("ImageCacheIt", () => require("nativescript-image-cache-it").ImageCacheIt);
```

**Comment by HoangJK** [comment]
Yes, I do it and the following markup:
`
<ImageCacheIt stretch="aspectFit"  centerCrop="false" resize="300,300" imageUri="http://screenrant.com/wp-content/uploads/The-Flash-vs-the-Reverse-Flash.jpg"></ImageCacheIt>
`
But it message error: TypeError: Could not load view for: ImageCacheIt.TypeError: Cannot read property 'id' of undefined

**Comment by triniwiz** [comment]
Ok i just published and update to fix that problem upgrade your package to `1.6.0`

**Comment by HoangJK** [comment]
Ok, I have done it. Thanks for your supporting!
